src/pages/browse.jsx

```diff
@@ -37,29 +37,42 @@ const ProgressIndicator = (
 );
 
 let fridgeList = null;
+const fridgePaperBoyLoveGallery = [40.697759, -73.927282];
+let prevFocusedMarker = null;
 export default function BrowsePage() {
   const [hasDataLoaded, setHasDataLoaded] = useState(false);
   const [currentView, setCurrentView] = useState(MapToggle.view.map);
+  const [currentMarker, setCurrentMarker] = useState(fridgePaperBoyLoveGallery);
 
   const availableHeight = useWindowHeight();
   const isWindowDesktop = useMediaQuery((theme) => theme.breakpoints.up('md'));
-
   useEffect(() => {
     const fetchData = async () => {
       fridgeList = await getFridgeList();
       setHasDataLoaded(true);
     };
     fetchData().catch(console.error);
   }, []);
+  // This useEffect is for closing the list view to focus the marker
+  useEffect(() => {
+    if (!isWindowDesktop && currentMarker !== prevFocusedMarker) {
+      prevFocusedMarker = currentMarker;
+      setCurrentView(MapToggle.view.map);
+    }
+    if (isWindowDesktop) {
+      prevFocusedMarker = currentMarker;
+    }
+  }, [currentMarker]);
 
   const Map = hasDataLoaded
     ? BrowseMap({
         fridgeList,
+        currentMarker,
       })
     : ProgressIndicator;
 
   const List = hasDataLoaded ? (
-    <BrowseList fridges={fridgeList} />
+    <BrowseList fridges={fridgeList} setCurrentMarker={setCurrentMarker} />
```

Comment:
As mentioned in another comment, passing the set functions is not a very clean practice. It looks like this setter is essentially drilled down through 2 children before it is used - which indicates the need for global state management.

Probably not much to be done about it currently without implementing a larger scale state management and refactoring a lot, but just noting for future reference.

Reply:
we can opt for zustand, but that's a choice for the right person to make